experimental/jvm/generate_projects.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Manager for running auto-gen from scratch."""
+
+import argparse
+import logging
+import os
+import shutil
+
+import utils
```

Comment:
Could you change this to a more specific import statement please?
E.g., `from experimental.jvm import utils`.
This statement is ambiguous for Python and will fail if there is a `util.py` in the dir with a higher search priority (e.g., the root of this repo).